.github/workflows/pull_request.yml

```diff
@@ -52,3 +54,22 @@ jobs:
           gcloud auth configure-docker ${{ secrets.GAR_LOCATION }}
           echo "Pushing to GAR with tag $TAG"
           docker push ${{ secrets.GAR_LOCATION }}/${{ secrets.GAR_PROJECT_ID }}/docker-images/eq-questionnaire-launcher:$TAG
+  lint:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '1.21'
+          cache: false
+      - name: Go linting
+        uses: golangci/golangci-lint-action@v4
+        with:
+          version: v1.54
+      - uses: actions/checkout@v3
+      - name: Install npm deps
```

Comment:
No need to install npm twice in this workflow, do what we did in Runner, something like:
```yaml
jobs:
  python-dependencies:
    runs-on: ubuntu-22.04
    steps:
      - uses: actions/checkout@v3
      - run: |
          echo "PYTHON_VERSION=$(cat .python-version)" >> $GITHUB_ENV
      - uses: actions/setup-python@v4
        with:
          python-version: ${{ env.PYTHON_VERSION }}
          cache: 'pipenv'
      - name: Install Pipenv
        run: pip install pipenv==2023.8.22
      - name: Install virtual environment
        run: |
          sudo apt-get install libsnappy-dev
          pipenv install --dev
  node-dependencies:
    runs-on: ubuntu-22.04
    steps:
      - uses: actions/checkout@v3
      - uses: actions/setup-node@v3
        with:
          node-version-file : ".nvmrc"
          cache: "npm"
      - name: Install npm deps
        run: npm install
  lint:
    needs: [python-dependencies, node-dependencies]
```

Reply:
Yep, we don't need to install it twice, updated 👍